listenbrainz/webserver/static/js/jsx/lastFmImporterModal.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import LastFmImporterModal from './lastFmImporterModal'
+
+let wrapper = null;
+describe('LastFmImporterModal is working correctly', () => {
```

Comment:
There's not much else to test on the modal, but you can check if close the button is disabled or enabled when expected when changing the props.